lib/api/endpoints.dart

```diff
@@ -8,4 +8,7 @@ class Endpoints {
 
   // extra sendContext
   static const String DO_NOT_SEND_CONTEXT = "&sendContextEvent=false";
+
+// Bucketing
+  static const String BucketingScript = "https://cdn.flagship.io/%s/bucketing.json";
```

Comment:
Case is not the same as the variable above, (CAMEL_CASE instead of CamelCase), is it normal ?